spring-xd-dirt/src/main/java/org/springframework/xd/dirt/plugins/stream/ModuleTypeConversionPlugin.java

```diff
@@ -142,14 +166,14 @@ private MimeType resolveContentType(String type, Module module) throws ClassNotF
 		return MimeType.valueOf(type);
 	}
 
-	private Class<?> resolveJavaType(String type, Module module) throws ClassNotFoundException, LinkageError {
+	private static Class<?> resolveJavaType(String type, Module module) throws ClassNotFoundException, LinkageError {
 		SimpleModule sm = (SimpleModule) module;
 		return ClassUtils.forName(type, sm.getApplicationContext().getClassLoader());
 	}
 
 	@Override
 	public boolean supports(Module module) {
-		return true;
+		return module.shouldBind();
```

Comment:
Ilaya created this JIRA to capture the post-1.1 refactoring:
https://jira.spring.io/browse/XD-2695